server/app/models/external_volume.rb

```diff
@@ -0,0 +1,8 @@
+class ExternalVolume
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :name, type: String
+  
+  has_one :volume
```

Comment:
This should be `belongs_to :volume` with a `has_many :external_volumes` on the `Volume` model.